00-HTML-CSS-basics/Punto 2/2/style.css

```diff
@@ -0,0 +1,96 @@
+!html {
+	font-size: 14px;
+	font-family: Arial;
+}
+
+header, footer, aside, nav {
+	background-color: yellow;
+}
+
+header, footer {
+	text-align: center;
+}
+
+.header {
+	font-size: 46px;
+	width: 100%;
+	box-sizing: border-box;
+	border-radius: 10px;
+	padding: 5px;
+	margin-bottom: 5px;
+	position: relative;
+	clear: both;
+}
+
+.content .header {
+	font-size: 14px;
+}
+
+.footer {
+	font-size: 10px;
+	box-sizing: border-box;
+	display: block;
+	clear: both;
+	width: 100%;
+	height: 50px;
+	border-radius: 10px;
+	padding: 5px;
+}
+
+.content {
+	background-color: yellow;
```

Comment:
Unify the way to use colors, if you use hexa use hexa for all of them, don't mix hexa, rgb and named colors.